### DISCUSSION TITLE
MinIO ILM

### DISCUSSION THREAD

**Discussion by SriRamJai8** [minio/minio#20248] [Q&A]
Hi 
I have configured MinIO with multiple Tier Targets, How is the data spread between all these targets ? is it roundrobin, least IO etc ? do we hav any option configure the algorithm ?
![image](https://github.com/user-attachments/assets/5819169a-07a5-4f88-afb1-c8375b405915)

**Comment by klauspost** [comment]
There is no "data spreading".

**Reply by SriRamJai8** [reply]
Thank you, so for a given bucket, even thought I have multiple tiers configured. will it destage to only 1 Target ?

**Reply by klauspost** [reply]
Yes.
